Exercises/4-count-types.js

```diff
@@ -1,5 +1,11 @@
 'use strict';
 
-const countTypesInArray = null;
+const countTypesInArray = array => {
+  const r = {};
+
+  for (const i of array) !r[typeof i] ? r[typeof i] = 1 : r[typeof i] += 1;
```

Comment:
Code duplication and looks unreadable.
See alternative implementations of other students (closed PRs).